_posts/2018-08-22-android-update.md

```diff
@@ -0,0 +1,43 @@
+---
+layout:     post
+title:      Servo for Android: nightly builds available and contribution opportunities
+date:       2018-08-22 09:00:00
+summary:    Servo for Android: nightly builds available and contribution opportunities
+categories:
+---
+
+We [recently revamped](https://github.com/servo/servo/pull/20912) Servo on Android.
```

Comment:
Maybe something about the motivation/why? Something like "As we continue to work on new implementation techniques and support for new platforms and technologies, we'd like to make this build available for testing and feedback."